When a test fails in the `beforeEach` the HTML reporter doesn't show the test failure message

(The dot reporter however displays it fine)

Code to reproduce:

```js
'use babel'

import {expect} from 'chai'

describe('Foo', () => {
  beforeEach(() => {
    expect(1).to.equal(2)
  })

  it('passes', () => {
    expect(1).to.be.ok
  })
})
```

Expected something approximating:

```
[etch][~/Source/soft-wrap-indicator] atom --test test
[4822:0627/170837:WARNING:resource_bundle.cc(311)] locale_file_path.empty() for locale English
[4822:0627/170837:WARNING:dns_config_service_posix.cc(146)] dns_config has unhandled options!
Error Downloading Update: Could not get code signature for running application

Window load time: 1082ms

  ․[4822:0627/170839:INFO:CONSOLE(462)] "Could not resolve 'soft-wrap-indicator' to a package path", source: /Users/Lee/Source/atom/src/package-manager.coffee (462)
․

  0 passing (385ms)
  2 failing

  1) Foo "before each" hook for "passes":

      AssertionError: expected 1 to equal 2
      + expected - actual

      -1
      +2

      at Context.<anonymous> (/Users/Lee/Source/soft-wrap-indicator/test/foo.test.js:5:18)
      at tryOnImmediate (timers.js:534:15)
      at processImmediate [as _immediateCallback] (timers.js:514:5)

  2) SoftWrapIndicatorPackage "before each" hook for "creates an indicator element":
     Error: Failed to load package 'soft-wrap-indicator'
      at PackageManager.module.exports.PackageManager.activatePackage (/Users/Lee/Source/atom/src/package-manager.coffee:460:26)
      at Context.<anonymous> (/Users/Lee/Source/soft-wrap-indicator/test/soft-wrap-indicator.test.js:16:28)
      at undefined.next (native)
      at step (/Users/Lee/Source/soft-wrap-indicator/test/soft-wrap-indicator.test.js:1:12)
```

Actual:

![screen shot 2016-06-27 at 5 10 14 pm](https://cloud.githubusercontent.com/assets/1038121/16399739/2bc2a04a-3c8a-11e6-811b-4138d429438e.png)

--- a/lib/html-reporter/components/test-failures.js
+++ b/lib/html-reporter/components/test-failures.js
@@ -41,12 +41,24 @@ export default class TestFailures {
         <div className='root'>
           <span className='header'>{suite.title}</span>
           <div className='tests-container'>
-            {this.renderTestFailures(suite.tests)}
-            {suite.suites.map(this.renderSubSuite.bind(this))}
+            {this.renderSuiteFailures(suite)}
           </div>
         </div>
       )
     }
+  }
+
+  renderSuiteFailures (suite) {
+    return (
+      <div>
+        {this.renderTestFailures(suite._beforeAll)}
+        {this.renderTestFailures(suite._beforeEach)}
+        {this.renderTestFailures(suite._afterEach)}
+        {this.renderTestFailures(suite._afterAll)}
+        {this.renderTestFailures(suite.tests)}
+        {suite.suites.map(this.renderSubSuite.bind(this))}
+      </div>
+    )
   }
 
   renderTestFailures (tests) {
@@ -73,8 +85,7 @@ export default class TestFailures {
         <div className='tests-container'>
           <span>{suite.title}</span>
           <div className='tests-container'>
-            {this.renderTestFailures(suite.tests)}
-            {suite.suites.map(this.renderSubSuite.bind(this))}
+            {this.renderSuiteFailures(suite)}
           </div>
         </div>
       )

